components/steps/w-step.vue

```diff
@@ -54,45 +54,42 @@ const getAriaLabel = (props) => {
 const stepClasses = computed(() => [
   ccStep.step,
   {
-    [ccStep.stepVertical]: vertical.value,
-    [ccStep.stepVerticalLeft]: vertical.value && left.value,
-    [ccStep.stepVerticalRight]: vertical.value && !left.value,
-    [ccStep.stepHorizontal]: !vertical.value,
+    [ccStep.vertical]: vertical.value,
+    [ccStep.alignLeft]: vertical.value && left.value,
+    [ccStep.alignRight]: vertical.value && !left.value,
+    [ccStep.horizontal]: !vertical.value,
   },
 ]);
 
-const horizontalClasses = computed(() => [
+const lineHorizontalClasses = computed(() => [
   ccStep.stepLine,
```

Comment:
This has been renamed to `line`, right? 🤔

Reply:
whoops! You're correct! I'll change it! 😄